dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -4,10 +4,15 @@
 class Api::V1::MlModelsController < ApplicationController
```

Comment:
if this API will always return JSON, you'll get some convenience features by having this class inherit from [`Api::V1::JsonApiController`](https://github.com/code-dot-org/code-dot-org/blob/staging-next/dashboard/app/controllers/api/v1/json_api_controller.rb) instead of `ApplicationController`